Gemfile.lock

```diff
@@ -1,79 +1,79 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.4.3)
-      actionpack (= 7.0.4.3)
-      activesupport (= 7.0.4.3)
+    actioncable (7.0.5)
```

Comment:
Please upgrade rails in a separate PR, this is a lot for one PR! :)

Reply:
Sorry I can remove that from this PR.